Code/GraphMol/AddHs.cpp

```diff
@@ -136,6 +136,22 @@ std::map<unsigned int, std::vector<unsigned int>> getIsoMap(const ROMol &mol) {
   return isoMap;
 }
 
+bool is_atom_n_pyrrol_like(const ROMol &mol, const Atom *atom) {
```

Comment:
The name of this keeps confusing me, so maybe we can change it.
I know the current naming is because the original special case was pyrroles, but we're beyond that and "pyrrole-like" sounds to me like it needs to be a heteroatom.

How about: `may_need_extra_H()`?